engine/src/io/data_provider/UriDataProvider.cpp

```diff
@@ -150,18 +150,45 @@ data_handle uri_data_provider::get_next(bool open_file) {
 				this->directory_uris = BlazingContext::getInstance()->getFileSystemManager()->list(target_uri);
 			}
 
-			std::string ender = ".crc";
+			// sometimes parquet directories have somes files that
+			// have not the same schema as the *.parquet files
+			// we don't want the data provider handle this ones
+			std::vector<std::string> ignored_suffixes{".crc", "_metadata", "_SUCCESS"};
```

Comment:
we should add ipython notebook checkpoints to this those get made everywhere

Reply:
.ipynb_checkpoints
are the files itmakmes

Reply:
Added. Thank you.

Reply:
Added, thank you.